### DISCUSSION TITLE
Youtube link for bot experiments

### DISCUSSION THREAD

**Discussion by antdogevervast** [OpenVoiceOS/ovos-buildroot#193] [Show and tell]
https://youtube.com/shorts/Su46hpFUjHE?si=-FGRJIuzjcp4yQfD
https://youtu.be/o1RnYjDFVM0?si=NgQWBfKtWSAGsZtO
https://youtube.com/shorts/OFA1PizTkv0?si=koVdrur7g9_EQMoX

**Comment by JarbasAl** [comment]
just saw this, nice!

we have a chat room just to show off our builds https://matrix.to/#/#openvoiceos-show-and-tell:matrix.org

**Comment by j1nx** [comment]
Thank you very much. Added them to our Show & Tell playlist on our Youtube channel.

https://www.youtube.com/playlist?list=PLdZ9a1TS1aA2CispHh_Pkx1lg30gizjlD
